<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DrawerSettings</class>
 <widget class="QDialog" name="DrawerSettings">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>370</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>523</width>
    <height>370</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>523</width>
    <height>370</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>2D Drawer Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="runge.qrc">
    <normaloff>:/Runge/Resources/16x16/wrench.png</normaloff>:/Runge/Resources/16x16/wrench.png</iconset>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <widget class="QGroupBox" name="groupBoxAxesMarks">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>181</width>
     <height>131</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="toolTip">
    <string>Show axes marks</string>
   </property>
   <property name="title">
    <string>Axes &amp;Marks</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;X Step</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buddy">
     <cstring>lineEditStepX</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditStepX">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Distance between X axis marks</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Y Step</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buddy">
     <cstring>lineEditStepY</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditStepY">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Distance between Y axis marks</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditMarkLength">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Axes marks length on screen and images (in pixels)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Len&amp;gth</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buddy">
     <cstring>lineEditMarkLength</cstring>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxAxes">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>131</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Show axes</string>
   </property>
   <property name="title">
    <string>&amp;Axes</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Top</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buddy">
     <cstring>lineEditTopLength</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditTopLength">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Top semi-axis length</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditRightLength">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Right semi-axis length</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Right</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buddy">
     <cstring>lineEditRightLength</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Bottom</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buddy">
     <cstring>lineEditBottomLength</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditBottomLength">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Bottom semi-axis length</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>41</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Left</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buddy">
     <cstring>lineEditLeftLength</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditLeftLength">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Left semi-axis length</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAdjustLengths">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Reset all axes lengths to fit current graph</string>
    </property>
    <property name="text">
     <string>Ad&amp;just</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAxesColor">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Set axes color</string>
    </property>
    <property name="text">
     <string>&amp;Color...</string>
    </property>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>417</x>
     <y>16</y>
     <width>91</width>
     <height>101</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxRescaleAfterRunningFromPoint">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>320</y>
     <width>291</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Rescale to fit everything after each new soluttion added</string>
   </property>
   <property name="text">
    <string>Rescale A&amp;fter Running From Point</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxHandles">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>230</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>&amp;Handles</string>
   </property>
   <widget class="QLineEdit" name="lineEditHandleDiameter">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Solution handles diameter on screen and images (in pixels)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Diameter</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buddy">
     <cstring>lineEditHandleDiameter</cstring>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxLineWidths">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>391</width>
     <height>71</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Assign width to lines, 0 means no width at any scale</string>
   </property>
   <property name="title">
    <string>Line &amp;Widths</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QLineEdit" name="lineEditAxesWidth">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Axes width on screen and images (in coordinate units)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Ax&amp;es</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buddy">
     <cstring>lineEditAxesWidth</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Sol&amp;utions</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buddy">
     <cstring>lineEditSolutionsWidth</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSolutionsWidth">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Solutions width on screen and images (in coordinate units)</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>groupBoxAxes</tabstop>
  <tabstop>pushButtonAdjustLengths</tabstop>
  <tabstop>pushButtonAxesColor</tabstop>
  <tabstop>lineEditLeftLength</tabstop>
  <tabstop>lineEditTopLength</tabstop>
  <tabstop>lineEditBottomLength</tabstop>
  <tabstop>lineEditRightLength</tabstop>
  <tabstop>groupBoxLineWidths</tabstop>
  <tabstop>lineEditAxesWidth</tabstop>
  <tabstop>lineEditSolutionsWidth</tabstop>
  <tabstop>groupBoxAxesMarks</tabstop>
  <tabstop>lineEditStepX</tabstop>
  <tabstop>lineEditStepY</tabstop>
  <tabstop>lineEditMarkLength</tabstop>
  <tabstop>lineEditHandleDiameter</tabstop>
  <tabstop>checkBoxRescaleAfterRunningFromPoint</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="runge.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DrawerSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>471</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DrawerSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>437</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>485</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
